pep0/output.py

```diff
@@ -26,15 +26,58 @@
 
 indent = u' '
 
+
+def escape(text):
+    text = text.replace('.', r'\.')
+    return text
+
+
 def write_column_headers(output):
     """Output the column headers for the PEP indices."""
-    column_headers = {'status': u'', 'type': u'', 'number': u'num',
-                        'title': u'title', 'authors': u'owner'}
-    print(constants.column_format % column_headers, file=output)
-    underline_headers = {}
+    column_headers = {'status': u'', 'type': u'type', 'number': u'num',
+                      'title': u'title', 'authors': u'owner'}
+    markups = {}
+    column_format = constants.column_format.replace('|', '')
     for key, value in column_headers.items():
-        underline_headers[key] = constants.text_type(len(value) * '-')
-    print(constants.column_format % underline_headers, file=output)
+        markups[key] = constants.text_type(len(value) * '=')
+    markup_line = (column_format % markups).strip()
+    print(markup_line, file=output)
+    print((column_format % column_headers).strip(), file=output)
+    print(markup_line, file=output)
+
+
+def write_table(columns, output, include_header=False):
+    if not columns:
+        return
+
+    lengths = [[] for x in columns[0]]
+    for column in columns:
```

Comment:
`for row in rows:` (I won't make any further comments that relate to the name change)